polyglot.js

```diff
@@ -2,27 +2,56 @@ class PolyGlot {
+				}
+		}
+
+		static get languages() {
+			switch(game.system.id.toLowerCase()){
+				default:
+					return CONFIG[game.system.id.toUpperCase()].languages;
+				case "wfrp4e":
+					let re = /Language \((.+)\)/i;
+					const pack = game.packs.get("wfrp4e.skills");
+					return pack.getIndex().then( itemList => {
+						let langs = {};
+						for (let item of itemList){
+							let found = item.name.match(re);
+							if(found && found.length > 1 && found[1].length >= 1){
+								langs[found[1]] = found[0];
+							}
+						}
+						Promise.resolve(langs);
```

Comment:
Do you need to do `Promise.resolve` here? Wouldn't just returning `langs` work the same?
Also, if you're doing async stuff, I prefer to make the function itself async and use await statements, rather than using `.then`.
Only use `.then` if you want something to happen asynchronously without interrupting the flow of the current function logic.